arch/arm/src/common/gnu/fork.S

```diff
@@ -78,7 +77,11 @@
  ****************************************************************************/
 
 	.globl	up_fork
+#ifdef __ghs__
+	.type	up_fork, $function
+#else
 	.type	up_fork, function
```

Comment:
ditto

Reply:
> ditto

same as the previous code, I have tried the solution, but cannot work, and report error